keps/sig-multicluster/2149-clusterid/README.md

```diff
@@ -487,6 +488,24 @@ free to use non-namespaced properties (e.g. `fingerprint`) as they see fit, but
 any shared tooling should use appropriately namespaced names.
 
 
+#### Property Ladder
+
+We foresee that there will be more properties that the community wants to adopt as "well known" properties. Thus, we want to define a process for adopting a property to be a well known property in the future:
+
+1. **Extension Property Stage**: A property is implemented as an extension property if the implementer has no intent to elevate it to a standard property. This property cannot use the reserved suffixes and is not required to be implemented by all implementers.
+
+2. **Standard Property Stage**: The implementor can propose to add a property to be a **standard** property which means if anyone implements the property with the same name, it must follow the same definition and rules described in the KEP. Not all implementers are required to implement the property.
+   The process for proposing a standard property is as follows:
+    * The property sponsor must present a discussion agenda with clear use cases and motivation at a Multi-cluster SIG meeting.
+    * The community will vote on the proposal, and if it receives approval from at least 2/3 of voters, the property becomes a standard property and is incorporated into the About API KEP.
+
+3. **Core Property Stage**: After a property has maintained standard status for at least 3 months and has 3 or more implementations, one can propose to elevate it to a **core** property which means it must be implemented by every implementer that implements the About API.
```

Comment:
I'm wondering if the "implementer" / "implementation" here is the right word. Because like I think this makes sense when you are using a cloud provider and it's integrated and you expect to have all those properties but I can easily see a scenario where you are not on a cloud provider but want to use certain project and you populate this with some kind of ArgoCD setup or whatever automation you use to provision/configure your cluster.  Maybe talking about either (not specifically here) of projects that integrate with certain properties or using a term about producer/consumer.

Reply:
I meant that right now there is only two required field that are very static and easy to define: clusterset name and cluster name. To declare that you don't necessarily need any cluster manager project to define those. Do we want to change that by having a ton more of properties in the "core property stage"/required list of properties, if yes to what extent?

Also I am assuming there could be some case where the "implementation" is not only one project like if you have multiple providers publishing some properties. Could be perhaps that you are using some OSS cluster manager but still relies on your cloud provider to publish some of the properties for instance or a mix of project interacting with each other somehow.